Geometry2d/testing/RectTest.cpp

```diff
@@ -1,6 +1,8 @@
 #include <gtest/gtest.h>
+
 #include <Geometry2d/Rect.hpp>
-#include <Constants.hpp>
+#include <Geometry2d/Segment.hpp>
+#include <rj_constants/constants.hpp>
```

Comment:
Again, why does geometry depend on constants?

Reply:
It uses `RobotRadius`...

Reply:
Um, wtf? We should just redefine a `constexpr double RobotRadius` in this package for testing purposes if we really care about that particular case, but these should be completely independent of what we're using them for.

Reply:
idk I didn't write this library lmao

Reply:
Made a todo

Reply:
Weird. Okay well it's got a TODO issue now, we should be fine.